components/web_server.rst

```diff
@@ -73,9 +73,9 @@ Configuration variables:
         # Example configuration entry
         web_server:
           port: 80
-          css_include: "../../../esphome-docs/_static/webserver-v1.min.css"
+          css_include: "/config/esphome/webserver-v1.min.css"
           css_url: ""
-          js_include: "../../../esphome-docs/_static/webserver-v1.min.js"
+          js_include: "/config/esphome/webserver-v1.min.js"
```

Comment:
This is very aimed at running a specific installation type of Home Assistant. Other users, might not have this folder.

Is it an idea to make it relative to the current path?

Ex: `css_include: "assets/webserver-v1.min.css"`

That works for any installation type of ESPHome (and Home Assistant).